frontend/app/src/store/balances/types.ts

```diff
@@ -253,7 +254,7 @@ export type BalanceSnapshotPayload = {
   assetIdentifier: string;
   amount: string;
   usdValue: string;
-  location: string;
+  location?: string;
```

Comment:
In which case is the location used?

Reply:
It is used to mark the balance with a location, that later will be used to adjust that location value automatically (this is process on frontend)

Reply:
Ok, so it is frontend only value then? is there any other way to pass it so that we don't keep it in the type that is supposed to go to the backend, for clarity's sake?

Reply:
hmm yeah let me see the code, I think I can separate that location type